### PR TITLE
Remove klever dependencies

### PR DESCRIPTION
**Pull request by vmordan**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -50,10 +50,10 @@ cil_arch="cil.xz"
 compiled_cil_arch="frama_c_cil.xz"
 
 # Repositories
-klever_repo="https://github.com/mutilin/klever.git"
+cvv_repo="https://github.com/vmordan/cvv.git"
```
Comment (PavelAndrianov) [inline Makefile:53]: Maybe, place it to gitlab.ispras?

File: Makefile [hunk 2]
```diff
@@ -50,10 +50,10 @@ cil_arch="cil.xz"
...
 benchexec_repo="https://github.com/sosy-lab/benchexec.git"
-cif_repo="https://github.com/ldv-klever/cif.git"
-cif_compiled_link="https://github.com/ldv-klever/cif/releases/download/v1.2/linux-x86_64-cif-1.2.tar.xz"
+cif_repo="https://github.com/ldv-cvv/cif.git"
```
Comment (PavelAndrianov) [inline Makefile:55]: Is the link correct? I got `404` error.

File: Makefile (hunk 1 above)
Comment (vmordan) [reply Makefile:53]: I think we can change repository later

Comment (vmordan) [reply Makefile]: Done

**Pull request merged**